packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -217,6 +233,31 @@ async function transformGraphQLSchema(context, options) {
     transformerList.push(new SearchableModelTransformer());
   }
 
+  const transformerConfigPath = path.join(resourceDir, TRANSFORM_CONFIG_FILE_NAME);
+  const customTransformersConfig = loadCustomTransformersConfig(transformerConfigPath, context);
+  const customTransformers =
+    (customTransformersConfig && customTransformersConfig.transformers
+      ? customTransformersConfig.transformers
+      : [])
+      .map((transformer) => {
+        const fileUrlMatch = /^file:\/\/(.*)\s*$/m.exec(transformer);
+        const modulePath = fileUrlMatch ? fileUrlMatch[1] : transformer;
+        // handle 'cannot find module'
+        try {
+          const imported = require(modulePath);
+          const CustomTransformer = imported.default;
+          return CustomTransformer.call({});
+        } catch (error) {
+          context.print.error(error);
```

Comment:
I'm torn between whether I think this should fail fast or quietly fail by ignoring.

Reply:
I thought about it and think we should fail fast here with a helpful error to help push users to do the right thing the first time. I can envision getting frustrated when my transformer isn't working even though the project is successfully compiling. Printing the error is useful but I may miss it (especially in a CI/CD environment).